src/colorizer/utils/url_utils.ts

```diff
@@ -45,6 +45,7 @@ enum UrlParam {
   OUTLIER_COLOR = "outlier-color",
   FILTERED_MODE = "filter-mode",
   FILTERED_COLOR = "filter-color",
+  OUTLINE_COLOR = "outline-color",
```

Comment:
I wonder if we should have a unit test to ensure uniqueness among all these values since they are used as the url params

Reply:
might be a typescript way of enforcing without unit tests

Reply:
Added a linting rule for this!